infrastructure/templates/core/application/backups/backupVault.bicep

```diff
@@ -21,4 +24,12 @@ module backupVaultModule '../../../common/components/data-protection/backupVault
   }
 }
 
+module resourceGroupReaderRoleAssignmentModule '../../../common/components/resource-group/roleAssignment.bicep' = if (deployBackupVaultReaderRoleAssignment) {
+  name: '${vaultName}ResourceGroupRoleAssignmentDeploy'
+  params: {
+    principalIds: [backupVaultModule.outputs.principalId]
+    role: 'Reader'
+  }
+}
```

Comment:
This seems like a very generic role to be adding to the backup vault here.

Do you know why it has to be 'Reader' role specifically, or happen to have a pointer to any documentation explaining why 'Reader' is necessary, and if so I'll take a quick read? Thanks.

Reply:
Ok good to know, thanks!